backend/src/game/PokerTable.ts

```diff
@@ -59,6 +59,19 @@ export default class PokerTable {
                     // Update the seat properties
                     seat.playerId = clientId;
                     seat.isTaken = true;
+                    // check if table is fully/ready
+                    const tableReady = this.checkTableReady(tableName)
```

Comment:
we already have the table so we could send that instead of the name and then in the function we don't need to call `getTable` again

Reply:
Love it